### ISSUE TITLE
Don't publish unnecessary BUILD files in the node package

### ISSUE THREAD

**Issue by payload** [grpc/grpc#12590]
I use version 1.4.1 of grpc.

I have a repository and build with bazel.
Also I have a dependency to grpc and install grpc with npm.
When I want to build my repository, bazel fails over BUILD files in node_modules/grpc

I expected no BUILD files in the published grpc node package.

**Comment by vjpai** [comment]
@murgatroid99  : did this get fixed after all?

**Comment by murgatroid99** [comment]
I think it was fixed when the referenced PR was merged, but it looks like it was unfixed by some changes to this repo.

**Comment by payload** [comment]
This PR might not fix this issue @murgatroid99 Did you checked that the published tar does not contain BUILD files? In 1.8.0 there are these BUILD files in grpc/deps
```
./grpc/deps/grpc/third_party/nanopb/BUILD
./grpc/deps/grpc/src/proto/grpc/testing/BUILD
./grpc/deps/grpc/src/proto/grpc/testing/duplicate/BUILD
./grpc/deps/grpc/src/proto/grpc/health/v1/BUILD
./grpc/deps/grpc/src/proto/grpc/status/BUILD
./grpc/deps/grpc/src/proto/grpc/reflection/v1alpha/BUILD
./grpc/deps/grpc/src/proto/grpc/core/BUILD
./grpc/deps/grpc/src/proto/grpc/lb/v1/BUILD
./grpc/deps/grpc/src/core/tsi/test_creds/BUILD
```

**Comment by murgatroid99** [comment]
It looks like that change was merged in after 1.8.0. We just haven't published it yet.

**Comment by murgatroid99** [comment]
We have now published 1.8.4, which no longer has those BUILD files.
